app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] it would be better to uncomment `confirmable` in order for the app will ask for a confirmation email as required 
<img width="334" alt="Screenshot 2022-12-22 at 17 53 50" src="https://user-images.githubusercontent.com/61790587/209186545-32290047-b8e7-4762-841c-fb5429fd50c2.png">